package.json

```diff
@@ -22,7 +21,8 @@
     "test:size": "bundlesize",
     "type-check": "tsc",
     "type-check:watch": "yarn type-check --watch",
-    "version": "yarn build"
+    "shipjs:prepare": "shipjs prepare",
+    "shipjs:release": "shipjs release"
```

Comment:
This is called every time when there's new commit pushed, but it triggers release only when a certain conditions are met.